substrate/primitives/version/src/lib.rs

```diff
@@ -182,6 +182,9 @@ pub struct RuntimeVersion {
 	/// between Wasm and native.
 	///
 	/// This number should never decrease.
+	///
+	/// This number is formatted as `u32` in a semver looking manner. For instance, `1.2.3` would
+	/// be encoded as `001_002_003`.
```

Comment:
```suggestion
```
Please don't change docs based on some higher level concept that are not related to this number.